tests/rptest/tests/usage_test.py

```diff
@@ -318,7 +318,11 @@ def delta_parse(x):
         fire_times = [delta_parse(x) for x in log_lines]
 
         # Ensure all deltas are 0 meaning there is no skew
-        return all([x == 0 for x in fire_times])
+        # Make an exception for 1 second delta to account for rounding skew
+        # when casting time now in epoch to seconds
+        return all(
+            [x == 0 or x == 1 for x in fire_times]
+        ), "A timer skew of greater then 1s has been detected within a usage fiber"
```

Comment:
You're returning a tuple, and now the `assert` will always pass because tuple is truthy.

You need to change the caller of this method to unpack the tuple, or move the error message to the assertion